server/src/main/resources/application.yml

```diff
@@ -92,13 +92,11 @@ check-ins:
     google-api:
       delegated-user: ${ GSUITE_SUPER_ADMIN }
       scopes:
-        scopeForDriveApi: "https://www.googleapis.com/auth/drive.file"
-        scopeForDirectoryApi: "https://www.googleapis.com/auth/admin.directory.user.readonly"
+        scope-for-drive-api: "https://www.googleapis.com/auth/drive.file"
+        scope-for-directory-api: "https://www.googleapis.com/auth/admin.directory.user.readonly"
```

Comment:
This is a no-op change, but kebab-case configuration is more idiomatic than camelCase